app/src/components/market/sections/market_create/steps/items/funding_and_fee_step.tsx

```diff
@@ -207,6 +247,20 @@ const FundingAndFeeStep: React.FC<Props> = (props: Props) => {
     setAllowanceFinished(false)
   }
 
+  const toggleCustomFee = () => {
+    if (customFee) resetTradingFee()
+    if (!customFee) setFee(undefined)
+    setCustomFee(!customFee)
+  }
```

Comment:
This could be clearer, maybe using an if-else instead of two if. Also `setCustomFee`  is always executed because there are no curly brackets.

```
const toggleCustomFee = () => {
  if(customFee) { 
     resetTradingFee()
     setCustomFee(false)
  } else {
    setFee(undefined)
    setCustomFee(true)
  }
}
```